common

```diff
@@ -72,7 +72,7 @@ declare -A fsck_types=([btrfs]=0    # btrfs doesn't need a regular fsck utility
                        [vfat]=1
                        [xfs]=1)
 
-out() { printf "$1 $2\n" "${@:3}"; }
+out() { printf "%s %s\n%s" "$1" "$2" "${@:3}"; }
```

Comment:
This is not applicable as mentioned. We use external format string here.

Reply:
Right - will rebase/fix once there's a bit more feedback && the pre-requirements have landed.